### DISCUSSION TITLE
Stats page disappeared

### DISCUSSION THREAD

**Discussion by valeriogalano** [skymethod/op3#31] [Q&A]
The following stats page worked until some days ago, but now is disappeared: https://op3.dev/show/7b5d2ff7-d9f8-572a-8ec5-e872cc46ba4c

Is possible to reactivate it?

Thanks you

**Comment by johnspurlock-skymethod** [comment]
Interesting, what's the feed url?

**Reply by valeriogalano** [reply]
Podcast feed: https://pensieriincodice.it/podcast/index.xml
Podcast name: Pensieri in codice

**Reply by johnspurlock-skymethod** [reply]
entering that feed url into https://op3.dev/setup leads to the stats page: https://op3.dev/show/200172f3-b27a-5d8b-87ec-94707d1dd8b4

I wonder why the podcast:guid changed (the bit after /show/ is the podcast guid), its not declared in the feed anyway - have you made any changes recently?

**Reply by valeriogalano** [reply]
New episode published on 25/04.

I think there is no podcast:guid tag into my feed. Should I specify it? Should I use 200172f3-b27a-5d8b-87ec-94707d1dd8b4 ?

**Reply by johnspurlock-skymethod** [reply]
It's not strictly necessary: if the feed doesn't specify one, the podcast index assigns you one based on your feed url

But yes, if you have the ability to add a tag to your feed, definitely add `<podcast:guid>200172f3-b27a-5d8b-87ec-94707d1dd8b4</podcast:guid>` at the channel level, since that seems to be the current one assigned to that feed.

This way if you move your feed to a new url later, apps can recognize this as the same show - see the [namespace docs for podcast:guid](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#guid) for more details

**Reply by valeriogalano** [reply]
Thanks @johnspurlock-skymethod!
It worked: https://op3.dev/show/7b5d2ff7-d9f8-572a-8ec5-e872cc46ba4c is now up and running again.
